modules/jt-examples-of-common-use-cases.adoc

```diff
@@ -50,7 +50,7 @@ node('maven') {
 
 ----
 
-.Example: Compile and build an image and deploy it to OpenShift using maven in Tekton.
+.Example: Compile and build an image and deploy it to OpenShift using maven in OpenShift Pipelines.
```

Comment:
If this is Apache Maven, then all instances of `maven` should be capitalized https://maven.apache.org/. Also, on first use it should use the whole product name (`Apache Maven`).

Reply:
Updated. Thanks.